rest/src/main/java/org/jboss/pnc/rest/trigger/BuildTriggerer.java

```diff
@@ -144,4 +170,56 @@ private BuildConfigurationAudited getLatestAuditedBuildConfiguration(Integer bui
         }
         return buildConfigRevs.get(0);
     }
+    
+    /*
+     * TODO: Do not ignore certificates, rather setup servers properly.
+     */
+    private void signalBpmEvent(String uri) {
```

Comment:
Agree to put it in common module. If it is not just few lines of code i would prefer having a client in its own classes.